nvflare/fuel/hci/server/hci.py

```diff
@@ -121,7 +141,7 @@ def __init__(
             ctx.load_verify_locations(ca_cert)
             ctx.load_cert_chain(certfile=server_cert, keyfile=server_key)
 
-            # replace the socket with an ssl version of itself
+            # replace the socket with a ssl version of itself
```

Comment:
In this case, I think "an ssl" is actually the right way because it depends on the sound of the following word.

Reply:
PyCharm tagged it as an error so I was just trying to shut it up.

Reply:
Reversed the change.